src/blocks/blocks/form/input/block.json

```diff
@@ -6,7 +6,7 @@
 	"category": "themeisle-blocks",
 	"description": "Display a contact form for your clients.",
 	"keywords": [ "input", "text", "email" ],
-	"parent": [ "themeisle-blocks/form" ],
+	"ancestor": [ "themeisle-blocks/form" ],
```

Comment:
@HardeepAsrani @JohnPixle @mghenciu  by using this, we can allow the other blocks (section/columns/ and everything with an inner block) to insert the inputs.

With this, we can do the multiple inputs on a row.

![image](https://user-images.githubusercontent.com/17597852/175492248-c40baa0d-7bed-44e1-9a3b-ab937df8a372.png)

The problem: this is a WP 6.0 feature. On the lower version, this field is ignored, and the inputs can be inserted from any block, not just those that are subordinated to the Form block.

Do you think this is a good compromise?

Reply:
We can just enable it for users on 6.0 and use the parent for below 5.9, I think that compromise is good enough for me. What do you think @JohnPixle?

I'm more concerned if you think this is the right way of having form designs by using Columns and other blocks inside. What do you think?

Reply:
I think for this release we can just allow it to use other blocks inside the form if a user is using 6.0 or above and leave it at that as you @mghenciu  and tackle the 50% issue is some other release. This way we can get this done for a release + make it possible to have that layout using the core columns.

In the future, it just might make sense to go with this as having a more complex layout system will be a pain in the CSS as we can't afford to update the HTML many times so using an outside block might be an easier way but we can figure that out some other time. 😄

Reply:
I also do not believe that using this method (sections) is the best way to handle the 50% issue, but on the other hand I understand the compromises we need to make for Gutenberg. 

As a user, there is no question that I would expect a more industry-standard way to manage field widths. This method feels like a hack :P

I understand Hardeep's concerns however. 
Still thinking..